pkg/wakaama/contrib/lwm2m_objects/device/device.c

```diff
@@ -27,6 +27,19 @@
 /* Set to true if reboot requested. */
 static bool reboot;
 
+/* Lookup table for static resources of device object */
+static const char *_static_resources[] = {
```

Comment:
The array element are not contiguous, so the array will have holes and be bigger than needed. A possible solution is to rearrange LWM2M_RES enum (btw, it is better for generating documentation if enums have a name) so that all static resources appear at the beginning.

Reply:
The enum order is not arbitrary, it corresponds to the ID of the resources given by the object description [here](http://www.openmobilealliance.org/tech/profiles/LWM2M_Device-v1_0_3.xml). I will add the name and improve the documentation for the values.